esp_pytest/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.7-buster
```

Comment:
Can we uninstall unused packages from this image? Or use `python:3.7-slim-buster` to get smaller image?